### DISCUSSION TITLE
ISR (Incremental Static Regeneration) with Cloudflare Pages

### DISCUSSION THREAD

**Discussion by rayblair06** [nitrojs/nitro#2568] [Q&A]
Has anyone investigated or found possible solutions for achieving ISR (Incremental Static Regeneration) with Cloudflare Pages?

We are exploring options to scale our Nuxt 3 SSG application because our estimated build times are quite high. We will have over 6000 pages, which may take around 30 minutes per build, and changes to our infrastructure might not be possible.

**Comment by Heziode** [comment]
Also searching for a solution, for a package manager website.
